docs/flink-configuration.md

```diff
@@ -133,7 +133,8 @@ env.getConfig()
-
+| watermark-column              | connector.iceberg.watermark-column              | N/A                          | null                             | Specifies the watermark column to use for watermark generation.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                    | 
+| watermark-timeunit            | connector.iceberg.watermark-timeunit            | N/A                          | TimeUnit.MICROSECONDS            | Specifies the watermark time unit to use for watermark generation.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                 | 
```

Comment:
Could we describe to possible values? What should the user write in the option string to get the desired result? Something like: `The possible values are 
DAYS, HOURS, MINUTES, SECONDS, MILLISECONDS, MICROSECONDS, NANOSECONDS`